.github/workflows/workflow.yml

```diff
@@ -154,6 +154,42 @@ jobs:
       - uses: cachix/install-nix-action@v18
       - run: nix develop .#fmt -c make fmt
 
+  coverage:
+    name: Coverage
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ocaml-compiler:
+          - 4.14.x
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use OCaml ${{ matrix.ocaml-compiler }}
+        uses: ocaml/setup-ocaml@v2
+        with:
+          ocaml-compiler: ${{ matrix.ocaml-compiler }}
+          opam-depext: false
+
+      - name: Set git user
+        run: |
+          git config --global user.name github-actions[bot]
+          git config --global user.email github-actions[bot]@users.noreply.github.com
+
+      # Install ocamlfind-secondary and ocaml-secondary-compiler, if needed
+      - run: opam install ./dune.opam --deps-only --with-test
+
+      - name: Install deps on Unix
+        run: |
+          opam install . --deps-only --with-test
+          opam exec -- make coverage-deps
+          opam exec -- make dev-deps
+          opam exec -- make coq-deps
```

Comment:
Can we exclude coq from coverage? Or have it exist in its own job.

Reply:
Absolutely.